src/main/java/bio/terra/service/tabulardata/google/bigquery/BigQueryDatasetPdao.java

```diff
@@ -1,19 +1,6 @@
 package bio.terra.service.tabulardata.google.bigquery;
 
-import static bio.terra.common.PdaoConstant.PDAO_DELETED_AT_COLUMN;
-import static bio.terra.common.PdaoConstant.PDAO_DELETED_BY_COLUMN;
-import static bio.terra.common.PdaoConstant.PDAO_FLIGHT_ID_COLUMN;
-import static bio.terra.common.PdaoConstant.PDAO_INGESTED_BY_COLUMN;
-import static bio.terra.common.PdaoConstant.PDAO_INGEST_DATE_COLUMN_ALIAS;
-import static bio.terra.common.PdaoConstant.PDAO_INGEST_TIME_COLUMN;
-import static bio.terra.common.PdaoConstant.PDAO_LOAD_HISTORY_STAGING_TABLE_PREFIX;
-import static bio.terra.common.PdaoConstant.PDAO_LOAD_HISTORY_TABLE;
-import static bio.terra.common.PdaoConstant.PDAO_LOAD_TAG_COLUMN;
-import static bio.terra.common.PdaoConstant.PDAO_ROW_ID_COLUMN;
-import static bio.terra.common.PdaoConstant.PDAO_TRANSACTIONS_TABLE;
-import static bio.terra.common.PdaoConstant.PDAO_TRANSACTION_ID_COLUMN;
-import static bio.terra.common.PdaoConstant.PDAO_TRANSACTION_STATUS_COLUMN;
-import static bio.terra.common.PdaoConstant.PDAO_TRANSACTION_TERMINATED_AT_COLUMN;
+import static bio.terra.common.PdaoConstant.*;
```

Comment:
We try to avoid `*` imports. I suspect IntelliJ did this.  You can disable this behavior in the code gen config

Reply:
Thank you! I didn't want to add this change myself, but didn't know how to disable the behavior.  Can you point me to where the codegen config lives?

Reply:
It's a little hacky but here's how I did it:
<img width="993" alt="image" src="https://user-images.githubusercontent.com/5633787/172454460-d15c2f72-81e0-4823-aa9f-f1b954331195.png">

In the code style/java settings of intellij, under the import tab, I set the class count ad names count fields to `9999`

Reply:
I appreciate the screenshot, thanks!

Reply:
If you enable `editorconfig` support you don't need to do this since the `.editorconfig` for this repo has this setting configured. To enable it, use `Editor > Code Style > Enable EditorConfig support`.